src/cli/key/sev/sign.rs

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::backend::sev::snp::launch::IdAuth;
+use crate::backend::ByteSized;
+
+use std::fmt::Debug;
+use std::fs::File;
+use std::io::prelude::*;
+
+use anyhow::{bail, Context};
+use camino::Utf8PathBuf;
+use clap::Args;
+use p384::ecdsa::signature::Signer as _;
+use p384::ecdsa::SigningKey;
+use p384::elliptic_curve::sec1::Coordinates;
+use p384::pkcs8::DecodePrivateKey;
+use p384::EncodedPoint;
+
+/// Generate SEV Signature for provided SEV keys and write to file.
+#[derive(Args, Debug)]
+pub struct Options {
+    /// SEV P-384 private key in PEM form
+    #[clap(long, required = true)]
```

Comment:
Is `required = true` doing anything here? Isn't it implied that all the arguments here required since they're not `Option`s?

Reply:
no, it's not.. thank you!